index.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html lang="ru">
+<head>
+	<meta charset="UTF-8">
+	<title>Document</title>
+	<link rel="stylesheet" href="style.css" type="text/css" />
+</head>
+<body>
+	
+	<div id="container">
+		<div id="start-container">
+			<div class="title">Для сортировки введите числа разделяя их запятой.</div>
+			<form class="form">
+			<input type="text" name="form" id="sort" class="input" placeholder="например: 1,3,10,884,3,4"/>
```

Comment:
I’ve heard that usage of id-s inside the component isn’t a good style. You can use css selector or mark needed element with data-attribute.